Stripe3DS2/Stripe3DS2/Resources/Stripe3DS2.xcassets/cartes-bancaires-logo.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "cartes-bancaires-logo.png",
```

Comment:
This seems like a new asset, is this correct?

Reply:
No, it existed in Stripe3DS2 before this.